frontend/src/routes/Bookshelf.jsx

```diff
@@ -2,22 +2,23 @@ import '../styles/Bookshelf.css';
 
 import { useState, useEffect } from 'react';
 import Container from 'react-bootstrap/Container';
-import DocumentsCards from '../components/DocumentsCards';
+import GraphContainer from '../components/GraphContainer';
 
 function Bookshelf({backend, user}) {
+  console.log('Bookshelf');
```

Comment:
Console logs are often necessary at development time but they should be cleaned away at integration time so that future developers won't get spammed.